site/app/templates/polls/ViewPoll.twig

```diff
@@ -63,10 +63,13 @@
                                 "content" : option.getResponse(),
                                 "style": "display: inline"
                             } only %}
+                            {% if option.isCorrect() and poll.isReleaseAnswer()%}
+                                <p id='correctTag' style="color: #0eed0e; margin-left: 0rem ">correct!</p>
```

Comment:
I personally think it looks better when it's capitalized personally, but I'm open to other opinions.  This is a suggestion, not a requested change.

```suggestion
                                <p id='correctTag' style="color: #0eed0e; margin-left: 0rem ">Correct!</p>
```

Reply:
I think this tag would fit more uniformly if it's either all capped or not capitalized at all - I'm leaving it as it is for now

Reply:
What about capitalizing the entire word and putting it in a green pill like the course materials page has?

As it is now, the text has poor contrast in light mode.  The pills on the course materials page were designed to fix this exact contrast issue.

![image](https://user-images.githubusercontent.com/16820599/222309934-16c719ee-9fb9-4b0c-afd3-d9ed9cbe2dc9.png)